src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -348,6 +352,13 @@ private void registerAutoCommands(){
 
     NamedCommands.registerCommand("SwitchRPMShoot", new SwitchRPMShootNEO(intakeShooter));
 
+    // NamedCommands.registerCommand("AutonRuinerShoot", new
+    // AutonRuinerShoot(intakeShooter));
+    // NamedCommands.registerCommand("IntakeAutonRuiner", new
+    // IntakeAutonRuiner(intakeShooter));
+    // NamedCommands.registerCommand("AutonRuinerShootAndIntake", new
+    // IntakeAutonRuiner(intakeShooter));
```

Comment:
Isn't this command important? I thought you were intaking and shooting notes out as you are going through the notes in the middle for the auto-ruiners.

Reply:
Whoops, accidentally commented it out when commenting out the 2 separate intake and shooter ruiner commands